web-ng/src/app/components/main-page/main-page.component.html

```diff
@@ -13,8 +13,14 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-
-<div *ngIf="(activeProject$ | async) as project">
-  <div style="height: 48px">{{ project.title[lang] }}</div>
-</div>
-<ground-map></ground-map>
+<mat-sidenav-container class="main-page-container">
+  <mat-sidenav #sidenav mode="over" position="end" [(opened)]="isSidePanelOpen">
```

Comment:
Nit: Can we move `mat-sidenav` after `mat-sidenav-content`? Seems to be more intuitive since `content` is the main page area, while `sidenav` is the secondary view area. It also appears second on screen if visually scanning the page from left to right.

Reply:
done